dle/search/templates/search/elastic/search.html

```diff
@@ -97,6 +97,11 @@ <h5>Select one of three options:</h5>
             <input type="radio" id="search-type-semantic" name="search-type" value="knn">
             <label for="search-type-semantic">Semantic (Vector AKNN)</label>
           </div>
+          <div class="modal-body">
+            <h5>Tooltips in a modal</h5>
```

Comment:
Is this test code? Does this do anything?

Reply:
just test. nothing here